config.go

```diff
@@ -1,25 +1,85 @@
 package main
 
+import (
+	"os"
+	"os/user"
+	"path/filepath"
+)
+
 // Config holds the configuration for ralph-cli.
 type Config struct {
 	Debug     bool
 	LogOutput string // e.g. logstash
 }
 
+// List of scripts that are bundled with ralph-cli (at this moment, only idrac.py).
+var bundledScripts = []string{"idrac.py"}
+
+// GetCfgDirLocation gets path to current user's home dir and appends ".ralph-cli" to it.
+func GetCfgDirLocation() (string, error) {
+	user, err := user.Current()
+	if err != nil {
+		return "", err
+	}
+	return filepath.Join(user.HomeDir, ".ralph-cli"), nil
+}
+
+// PrepareCfgDir creates ~/.ralph-cli dir with its subdirs and copies bundled
+// scripts to "scripts" dir.
+func PrepareCfgDir() error {
+	var err error
+	loc, err := GetCfgDirLocation()
+	if err != nil {
+		return err
+	}
+	err = createCfgDir(loc)
+	if err != nil {
+		return err
+	}
+	var scriptsDir = filepath.Join(loc, "scripts")
+	// Copy bundled scripts.
+	for _, script := range bundledScripts {
+		if _, err := os.Stat(filepath.Join(scriptsDir, script)); os.IsNotExist(err) {
+			err = RestoreAsset(scriptsDir, script)
+			if err != nil {
+				return err
+			}
+		}
+	}
+	return nil
+}
+
+// createCfgDir is a helper function for creating ~/.ralph-cli dir (used by
+// PrepareCfgDir).
+func createCfgDir(loc string) error {
```

Comment:
Yes, names starting with lower-case letter are unexported ("private") in Go. It doesn't make much sense here though, since there's only one package at this moment (`main`), and the whole program is meant to be used as an executable, not a library, but in the future I'd like to keep this function private, so that's the reason why I did it.